app/presenters/download_filename_helper.rb

```diff
@@ -43,24 +43,18 @@ def self.filename_for_asset(asset, derivative: nil)
     self.filename_with_suffix(base, content_type: content_type)
   end
 
-  # Try to keep the ris download filename similar to the one we use
-  # for originals: filename_for_asset(asset).
-  def self.ris_download_name(work)
-    base = if work.representative.nil?
-      first_three_words(work.title)
-    else
-      self.filename_base_for_asset(work.representative)
-    end
-    "#{base}.ris"
-  end
-
   def self.filename_base_for_asset(asset)
+    raise ArgumentError, 'Pass in an asset.' unless asset.is_a? Asset
     if asset.content_type && asset.content_type.start_with?("audio/")
       return asset.title
     end
     self.filename_base_from_parent(asset)
   end
 
+  def self.ris_download_name(work)
+    "#{first_three_words(work.title)}.ris"
+  end
```

Comment:
For other downloads, in addition to first three words, we put the friendlier_id in, just as a unique ID/disambiguation in case other things have same first three words. 

Based on your report-back of what chf_sufia does, it looks like chf_sufia does that too? So keep this consistent for now, just porting behavior over, and add friendlier_id too?